Makefile

```diff
@@ -41,25 +42,33 @@ build-docs:
 	$(MAKE) -C docs clean
 	$(MAKE) -C docs html
 	$(MAKE) -C docs doctest
+
+validate-docs: build-docs
 	./newsfragments/validate_files.py
-	towncrier --draft --version preview
+	towncrier build --draft --version preview
```

Comment:
I guess I didn't realize this was a `build` command as well... I'm noticing as I look across the libraries that I don't think it makes a difference but maybe at some point using `build` vs not using `build` might end up with different behaviors. Something to keep in mind I suppose.

Reply:
Without the `build`, it looks like it just returns the towncrier version. I'll keep an eye out and update when I come across those elsewhere!